raster/src/main/scala/geotrellis/raster/stitch/Stitcher.scala

```diff
@@ -33,3 +33,37 @@ object Stitcher {
   }
 
 }
+
+
+trait StitcherR[T <: CellGrid, R <: CellGrid] extends Serializable {
```

Comment:
You're totally correct. I saw this problem and was a little afraid of how prevalent it was with our use of type classes and not having a solution for it. In retrospect we can just change the test and roll this in as it's own PR.